### PR TITLE
String formatting clean up.

### PR DESCRIPTION
**Pull request by xpconanfan**
* Use single quote by default.
* Remove usage of '.format'.

### CODE REVIEW

File: mobly/config_parser.py [hunk 1]
```diff
@@ -80,15 +80,15 @@ def _validate_testbed_configs(testbed_configs):
         _validate_testbed_name(name)
         # Test bed names should be unique.
         if name in seen_names:
-            raise MoblyConfigError("Duplicate testbed name %s found." % name)
+            raise MoblyConfigError('Duplicate testbed name %s found.' % name)
         seen_names.add(name)
 
 
 def _verify_test_class_name(test_cls_name):
```
Comment (adorokhine) [inline mobly/config_parser.py]: Is this method still used? I thought it was part of the old autodiscover codepath. (please don't update this in this PR though)

Comment (xpconanfan) [inline mobly/config_parser.py]: Do we want to enforce test class naming convention going forward?
Or do we just want to rely on class inheritance relationship

Comment (adorokhine) [inline mobly/config_parser.py]: I don't see any reason to enforce a test class naming convention. Any advantages?

Comment (xpconanfan) [inline mobly/config_parser.py]: Yeah, not much advantage other than making it easier to id which one is test for humans.
Example:
```
class Hahaha(Hehehe):
...
class Hehehe(base_test.BaseTestClass):
...
```
Not super critical

**Pull request merged**